apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/TableSchemaBuilderTest.cs

```diff
@@ -79,6 +80,50 @@ public void ValidateFieldName_Invalid(string name)
             Assert.Throws<ArgumentException>(() => TableSchemaBuilder.ValidateFieldName(name, "field"));
         }
 
+        [Fact]
+        public void AddWithPolicyTags()
+        {
+            var builder = new TableSchemaBuilder
+            {
+                { "field", BigQueryDbType.Int64, BigQueryFieldMode.Repeated, "My field" }
+            }.ModifyField("field", field => field.PolicyTags = new PolicyTagsData { Names = new[] { "policyName" } });
+
+            var schema = builder.Build();
+            var field = schema.Fields[0];
+            Assert.Equal("field", field.Name);
+            Assert.Equal("INTEGER", field.Type);
+            Assert.Equal("REPEATED", field.Mode);
+            Assert.Equal("My field", field.Description);
+            Assert.Contains("policyName", field.PolicyTags.Names);
+        }
+
+        [Fact]
+        public void AddPolicyTagWithNested()
+        {
+            var nested = new TableSchemaBuilder { { "inner", BigQueryDbType.String, BigQueryFieldMode.Nullable, "My field" } }
```

Comment:
I think the format of the previous test was cleaner here.

Reply:
removed this test.